src/main/java/matcha/storage/Storage.java

```diff
@@ -0,0 +1,105 @@
+package matcha.storage;
+
+import matcha.parser.Parser;
+
+import matcha.task.Task;
+
+import matcha.exception.MatchaException;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+
+
+/**
+ * Represents a Storage object that saves and loads tasks from a file.
+ */
+public class Storage {
+    private static String FILE_PATH;
```

Comment:
Since the variable is not final, perhaps it would be better to name it in camelCase instead